apps/site/components/withNavBar.tsx

```diff
@@ -19,12 +19,15 @@ import { useSiteNavigation } from '#site/hooks';
 import { useRouter, usePathname } from '#site/navigation.mjs';
 import { availableLocales } from '#site/next.locales.mjs';
 
-const SearchButton = dynamic(() => import('#site/components/Common/Search'), {
-  ssr: false,
-  loading: () => (
-    <Skeleton className={styles.searchButtonSkeleton} loading={true} />
-  ),
-});
+const SearchButton = dynamic(
+  () => import('#site/components/Common/Searchbox'),
+  {
+    ssr: false,
+    loading: () => (
+      <Skeleton className={styles.searchButtonSkeleton} loading={true} />
+    ),
+  }
+);
```

Comment:
Can this be SSRed now?
```suggestion
import SearchButton from '#site/components/Common/Searchbox';
```